webauthn/createcredential-timeout.https.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>WebAuthn navigator.credentials.create() timeout Tests</title>
+<link rel="author" title="Adam Powers" href="mailto:adam@fidoalliance.org">
+<link rel="help" href="https://w3c.github.io/webauthn/#iface-credential">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src=helpers.js></script>
+<body></body>
+<script>
+standardSetup(function() {
+    "use strict";
+
+    // bad timeout values
+    new CreateCredentialsTest({path: "options.publicKey.timeout", value: -1}).runTest("Bad timeout: negative", new TypeError());
+    new CreateCredentialsTest({path: "options.publicKey.timeout", value: 4294967295 + 1}).runTest("Bad timeout: too big", new TypeError());
```

Comment:
The spec doesn't impose an upper limit on the timeout size; I don't know if this is a valid test to assume it should be a TypeError. (Step 4 of [Section 5.1.3](https://w3c.github.io/webauthn/#createCredential))

Reply:
The `timeout` property is defined as `unsigned long`. The value is MAX_UNSIGNED_LONG + 1. I'll comment it out with a TODO while I research the proper behavior. :)

Reply:
Sounds good. That one might be tricky!

Reply:
fixed in 60f7ad1